FirstTask.py

```diff
@@ -0,0 +1,219 @@
+        self.account_no = account_no
+        self.account_type = account_type
+        self.password = password
+
+    def account_number(self):
+        return self.account_no
+
+    def pass_code(self):
+        return self.password
+    
+    def check_balance(self):
+        return self.balance
+
+    def check_account_type(self):
+        return self.account_type
+    
+    def deposit_bill(self, bill):
+        if bill >= 0:
+            self.balance += bill
+            return True
+        else:
+            return False
+    
+    def draw_money(self, money):
+        if self.balance >= money:
+            self.balance -= money
+            return True
+        else:
+            return False
+    
+    def account_details(self):
+        print("Account # ", self.account_no)
+        print("Name: ", self.name)
+        print("Balance: ", self.balance)
+        print("Account Type: ", self.account_type)
+  
+class Bank:
+
+    account_list = []
+
+    def create_account(self, account):
+        self.account_list.append(account)
+    
+    def login(self, account_number, code):
+        user1=self.find_account(account_number)
+        global logout_check
+        if  logout_check == False:
+            if user1[0]:
+                if code == user1[1].pass_code():
+                    logout_check = False
+                    return True, user1[1]
+            return False, "Account Not Found!"
+
+    def transfer_money(self, sender_account_number, receiver_account_number, amount):
+        user1 = self.find_account(sender_account_number)
+        user2 = self.find_account(receiver_account_number)
+        if user1[0]:
+            user2[1].deposit_bill(amount)
+            user1[1].balance -= amount
+            return True
+        else:
+            return False
+    
+    def compare(self,current_account,other_account):
+        user1 = self.find_account(current_account)
+        user2 = self.find_account(other_account)
+        if user1[0] and user2[0]:
+            if user1[1].check_balance() > user2[1].check_balance():
+                return True, 1
+            if user1[1].check_balance() < user2[1].check_balance():
+                return True, -1
+            if user1[1].check_balance() == user2[1].check_balance():
+                return True, 0
+        else:
+            return False, "Account Not Found!"
+    
+    def find_account(self, account_number):
+        for i in range(len(Bank.account_list)):
+            if account_number == Bank.account_list[i].account_number():
+                return True, Bank.account_list[i]
+        return False, "Account not found!"
+
+    def account_details(self, account_number):
+        check = self.find_account(account_number)
+        if check[0]:
+            check[1].account_details()
+        else:
+            return False
+   
+    def logout():
+        global logout_check
+        logout_check = True
+
+def run():
+    account_number_id = 1000
+    obj = []

```

Comment:
We should not be maintaining these here. The whole point of making the `account_list` in `Bank` class is to not manage it here.